pydantic/env_settings.py

```diff
@@ -244,3 +240,21 @@ def read_env_file(
         return {k.lower(): v for k, v in file_vars.items()}
     else:
         return file_vars
+
+
+def _read_env_files(
+    env_files: DotenvType, *, encoding: Optional[str], case_sensitive: bool
+) -> Dict[str, Optional[str]]:
+    if isinstance(env_files, (str, os.PathLike)):
+        env_files = [env_files]
+
+    dotenv_vars = {}
+    for env_file in reversed(env_files):
+        env_path = Path(env_file).expanduser()
+        if not env_path.is_file():
+            continue
+
+        for k, v in read_env_file(env_path, encoding=encoding, case_sensitive=case_sensitive).items():
```

Comment:
why not

```py
if env_path.is_file():
    dotenv_vars.update(read_env_file(env_path, encoding=encoding, case_sensitive=case_sensitive))
```
?

Reply:
I completely forgot `update` method. thanks!